src/main/java/com/chartiq/finsemble/example/MultiWindowJavaSwingExample.java

```diff
@@ -0,0 +1,54 @@
+package com.chartiq.finsemble.example;
+
+import it.sauronsoftware.junique.AlreadyLockedException;
+import it.sauronsoftware.junique.JUnique;
+
+import java.util.Arrays;
+
+
+public class MultiWindowJavaSwingExample {
+
+    // JUnique has a bug in which if you pass a non-unicode char, the recreation of the string from
+    // a inputstream gets messed up - therefore we have chosen to use & has the delimiter
+    // similar to what gets used in a query params URL
+    private static final String DELIMITER = "&";
```

Comment:
I think it would be better to have the delimited as a somewhat random multi-character string, so it is less likely to appear in an argument string.

Reply:
Updated delimiter to `!&!`